src/main/java/io/kamax/matrix/room/RoomTag.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * matrix-java-sdk - Matrix Client SDK for Java
+ * Copyright (C) 2018 Arne Augenstein
+ *
+ * https://www.kamax.io/
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package io.kamax.matrix.room;
+
+public class RoomTag {
+
+    private String namespace;
+    private String name;
+    private Double order;
+    private String roomId;
+
+    public RoomTag(String namespace, String name, Double order, String roomId) {
+        this.namespace = namespace;
+        this.name = name;
+        this.order = order;
+        this.roomId = roomId;
+    }
+
+    public String getNamespace() {
+        return namespace;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public Double getOrder() {
```

Comment:
We might want to use `Optional<Double>` as return type on this

Reply:
I think the order is mandatory as it has to be in the range of `[0,1]`.

Reply:
`order` is not marked as **Required** in the spec, and is therefore optional - unless you can show me something(s) that shows it should really be required.